basic-algorithm-scripting/boo-who.js

```diff
@@ -0,0 +1,9 @@
+function booWho(bool) {
+  if (bool === true || bool === false) {
```

Comment:
This logic is not good with a lot of redundancies
Instead of test if bool is true or false remove this and use typeof is boolean